### PR TITLE
Update mergeAOSP.sh

### PR DESCRIPTION
**Pull request by pjBooms**
- ignore AOSP stub folders that do not exist in CMP

### CODE REVIEW

File: scripts/mergeAOSP.sh [hunk 1]
```diff
@@ -59,4 +59,10 @@ resetSubFolder "./compose/**/desktopMain/**"
 resetSubFolder "./compose/**/skikoTest/**"	
 resetSubFolder "./compose/**/desktopTest/**"	
 resetSubFolder "./compose/desktop/**"	
+# specify folders that doesn't exist in JetBrains fork, because Git might treat some files as renames to these folders
+resetSubFolder "./compose/**/linuxx64StubsMain/**"
+resetSubFolder "./compose/**/linuxx64StubsTest/**"
+resetSubFolder "./compose/**/jvmStubsMain/**"
+resetSubFolder "./compose/**/jvmStubsTest/**"
```
Comment (MatkovIvan) [inline scripts/mergeAOSP.sh]:
```suggestion
resetSubFolder "./compose/**/*StubsMain/**"
resetSubFolder "./compose/**/*StubsTest/**"
```

File: scripts/mergeAOSP.sh [hunk 2]
```diff
@@ -59,4 +59,10 @@ resetSubFolder "./compose/**/desktopMain/**"
...
+resetSubFolder "./compose/**/nonAndroidMain/**"
```
Comment (MatkovIvan) [inline scripts/mergeAOSP.sh:65]: It's not part of stabs, please move it into appropriate place/section

**Review by igordmn — APPROVED** [review]

File: scripts/mergeAOSP.sh (hunk 2 above)
Comment (pjBooms) [reply scripts/mergeAOSP.sh:65]: The comment says about subfolders that does not exist in fork not about stubs

File: scripts/mergeAOSP.sh (hunk 1 above)
Comment (pjBooms) [reply scripts/mergeAOSP.sh]: fixed

**Pull request merged**